qualtran/bloqs/block_encoding/chebyshev_polynomial_test.py

```diff
@@ -130,16 +125,15 @@ def test_scaled_chebyshev_poly_odd(bloq_autotester):
     bloq_autotester(_scaled_chebyshev_poly_odd)
 
 
-@pytest.mark.slow
-def test_scaled_chebyshev_even_tensors():
-    from_gate = t4((XGate().tensor_contract() + Hadamard().tensor_contract()))
+def test_scaled_chebyshev_poly_even_tensors():
+    from_gate = t2(XGate().tensor_contract() * 3.14)
```

Comment:
Would be good to have a test where the input block-encoding doesn't satisfy $U^2 = I$. Perhaps using `MatrixGate.random`?